react/components/ChannelContents/components/ChannelContentsAppended/index.js

```diff
@@ -0,0 +1,75 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { withApollo } from 'react-apollo';
+import sharify from 'sharify';
+
+import initPusherClient from 'react/util/initPusherClient';
+
+import channelContentsSetQuery from 'react/components/ChannelContents/components/ChannelContentsSet/queries/channelContentsSet';
+
+import ChannelContentsSet from 'react/components/ChannelContents/components/ChannelContentsSet';
+
+const { data: { NODE_ENV } } = sharify;
```

Comment:
If testing this locally you have to be mindful of the environment you're running in vs the server's.

Reply:
By mindful I mean you won't be subscribed to the correct channels.